packages/app/src/cli/utilities/app/http-reverse-proxy.ts

```diff
@@ -67,14 +53,38 @@ export async function runConcurrentHTTPProcessesAndPathForwardTraffic(
 
   const availablePort = portNumber ?? (await port.getRandomPort())
```

Comment:
Probably worth a verbose mode output here with the `rules` mapping and the `availablePort`